common/test/acceptance/fixtures/catalog.py

```diff
@@ -13,31 +13,31 @@ class CatalogFixture(object):
     """
     Interface to set up mock responses from the Catalog stub server.
     """
-    def install_programs(self, programs):
+    def install_programs(self, data):
         """Set response data for the catalog's course run API."""
         key = 'catalog.programs'
 
-        requests.put(
-            '{}/set_config'.format(CATALOG_STUB_URL),
-            data={key: json.dumps(programs)},
-        )
+        if isinstance(data, dict):
+            key += '.' + data['uuid']
```

Comment:
NIT: and maybe this is just too old school way of thinking.  How often are we in here appending strings? Would it be better to do something like:
    key = ['catalog','programs']
then key.append(data['uuid'])
and round it off with:
    data={'.'.join(key): json....)

I see a lot of += and + on strings and makes me wonder if it's the right approach.